jupyterhub/jupyterhub/base/jupyterhub-dc.yaml

```diff
@@ -91,18 +91,43 @@ spec:
             configMapKeyRef:
               name: jupyterhub-cfg
               key: notebook_destination
-        image: quay.io/odh-jupyterhub/jupyterhub-img:v0.3.1
+        - name: TRAEFIK_API_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: $(traefik_credentials_secret)
+              key: TRAEFIK_API_USERNAME
+        - name: TRAEFIK_API_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: $(traefik_credentials_secret)
+              key: TRAEFIK_API_PASSWORD
+        image: quay.io/odh-jupyterhub/jupyterhub-img:pr-104
         imagePullPolicy: "Always"
         name: jupyterhub
         ports:
         - containerPort: 8080
           protocol: TCP
         resources:
           limits:
+            cpu: "1"
+            memory: 1Gi
+          requests:
+            cpu: "1"
             memory: 1Gi
         volumeMounts:
         - mountPath: /opt/app-root/configs
           name: config
+        readinessProbe:
+            tcpSocket:
+              port: 8081
+            initialDelaySeconds: 15
+            periodSeconds: 10
+      - name: jupyterhub-ha-sidecar
+        image: gcr.io/google_containers/leader-elector:0.5
+        args:
+        - --election=jupyterhub-ha-election
+        - --election-namespace=$(namespace)
```

Comment:
Would it make more sense to add namespace from metadata into an env var and use it in ARGS to avoid adding another place to template in `params.yaml`?